homework/hackyourtemperature/server.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+import express from 'express';
+import bodyParser from 'body-parser';
+
+let Server=express();
```

Comment:
```suggestion
const server = express();
```

Uppercase names are usually reserved for classes. Instances are typically lowercase. Also, our variable never changes so better to use `const` here.